xdsl/dialects/llvm.py

```diff
@@ -1735,56 +1734,132 @@ def __init__(self, val: Operation | SSAValue, res_type: Attribute):
         )
 
 
+class AbstractFloatArithOp(IRDLOperation, ABC):
+    T: ClassVar = VarConstraint("T", BaseAttr(AnyFloat))
+
+    lhs = operand_def(T)
+    rhs = operand_def(T)
+    res = result_def(T)
+
+    fastmathFlags = opt_attr_def(FastMathAttr)
```

Comment:
I just checked and this should be a property, non-optional, and have a default value

Reply:
It's marked as an attribute on [the docs](https://mlir.llvm.org/docs/Dialects/LLVM/#llvmfmul-llvmfmulop) though? And the format it prints in is also `{fastmathFlags = ...}` without the prop brackets...

Reply:
I'm sure we've had this exact conversation before, it literally prints with `<{...}>`
```
"builtin.module"() ({
  %0:2 = "test.op"() : () -> (f32, f32)
  %1 = "llvm.fmul"(%0#0, %0#1) <{fastmathFlags = #llvm.fastmath<none>}> : (f32, f32) -> f32
}) : () -> ()
```

Reply:
Everything is marked as an attribute on the docs, they never switched the docs over to properties